testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -14527,6 +14527,42 @@ private static void checkBoolAndFunc(SqlOperatorFixture f) {
     f.checkAgg("bool_and(x)", values4, isNullValue());
   }
 
+  @Test void testBoolAndAggFunc() {
```

Comment:
suggestion: can be merged with `SqlOperatorTest#testBoolAndFunc` to verify the same underlying Function implementation. 
Please refer to `SqlOperatorTest#testRlikeOperator`